internal/cephfs/volume.go

```diff
@@ -30,22 +30,35 @@ import (
 var (
 	// clusterAdditionalInfo contains information regarding if resize is
 	// supported in the particular cluster and subvolumegroup is
-	// created or not.
-	// Subvolumegroup creation and volume resize decisions are
+	// created or not. Also the cluster support `subvolume info` command..etc
```

Comment:
where is subvolumeinfo stored here?

Reply:
inside getSubvolumeInfo() we store that flag.